install-build-deps.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -o pipefail
+set -o errexit
+
+installdeps_fedora() {
+        sudo dnf install \
+        bats \
+        jq \
+        lxc-devel \
+        libcap-devel \
+        libacl-devel
+    if ! command -v go 2>/dev/null; then
+        sudo dnf install golang
+        go version
+    fi
+}
+
+installdeps_ubuntu() {
+    sudo add-apt-repository -y ppa:project-machine/squashfuse
```

Comment:
Hm - don't you need an apt-get update first, or does add-apt-repository do that for you these days?

Reply:
it does that for you now